### ISSUE TITLE
CST - Heading order is invalid for "Claim and Appeal status is unavailable" alert header

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/va.gov-team#108002]
## Issue
The H4 that shows under the Status tab component is out of order. It goes H1 -> H2 -> H4. It should be an H3 instead of H4.

New Cypress Axe rules for headings are coming and will be put into effect April 30. [Slack thread](https://dsva.slack.com/archives/C03R5SBELQM/p1742920275472879) The link to the Axe report expired, but I noticed this one while running an Axe Devtools scan.

Page: https://staging.va.gov/track-claims/your-claims

Doesn't look like a design system component, but need to check.

<img width="1380" alt="Image" src="https://github.com/user-attachments/assets/abcb4afe-681a-4f8d-874a-4a86be8f935e" />
